### DISCUSSION TITLE
Forgot password option for externally managed auth

### DISCUSSION THREAD

**Discussion by dragoangel** [grafana/grafana#64286] [General]
When Grafana used with external auth it's good have option to have Forgot password button on login page hidden or lead to custom url, but there no such option available.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I must be misunderstanding, but I think it would be up to your external auth provider to provide forgotten password capabilities. Feel free to make an [issue](https://github.com/grafana/grafana/issues) defining what youre looking for to request this feature.

**Reply by dragoangel** [reply]
Hi @gelicia , to get to external provider from grafana you must provide a link to it, right? This what ticket about:
1. Have option to hide forgot button if there no option to reset password in any possible way
2. If there an external forgot password available - set it in Grafana, so people can get to it easily
